contrib/pg_tde/documentation/docs/architecture.md

```diff
@@ -330,19 +330,21 @@ Changing the default principal key will rotate the encryption of internal keys f
 
 ### Removing key (not yet implemented)
 
-`pg_tde_drop_principal_key` removes the principal key for the current database.
+`pg_tde_drop_key` removes the principal key for the current database.
 If the current database has any encrypted tables, and there isn't a default principal key configured, it reports an error instead.
 If there are encrypted tables, but there's also a global default principal key, internal keys will be encrypted with the default key.
 
 It isn't possible to remove the WAL (server) principal key.
 
 ### Current key details
 
-`pg_tde_principal_key_info()` returns the name of the current principal key, and the provider it uses.
+`pg_tde_key_info()` returns the name of the current principal key, and the provider it uses.
 
-`pg_tde_global_principal_key_info(‘PG_TDE_GLOBAL')` does the same for the server key.
+`pg_tde_server_key_info('PG_TDE_GLOBAL')` does the same for the server key.
```

Comment:
`PG_TDE_GLOBAL` does not exist anymore.

Reply:
This function doesn't have arguments at all, so removed.